nan_typedarray_contents.h

```diff
@@ -0,0 +1,82 @@
+// Copyright Joyent, Inc. and other Node contributors.
```

Comment:
I just copy-pasted the boilerplate from nan.h.  If you have an alternative license I should use, let me know.